### PR TITLE
Update release notes draft to version v1.26.0-alpha.2

### PR DESCRIPTION
**Pull request by harshanarayana**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.26.0-alpha.2

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

File: releases/release-1.26/release-notes/maps/pr-112607-map.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pr: 112607
+releasenote:
+  text: Consider only plugin directory and not entire kubelet root when cleaning up
```
Comment (laxmikantbpandhare) [inline releases/release-1.26/release-notes/maps/pr-112607-map.yaml:3]: past tense

`Considered only plugin directory and not entire kubelet root when cleaning up`

File: releases/release-1.26/release-notes/maps/pr-112741-map.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+pr: 112741
+releasenote:
+  text: Expose health check SLI metrics on `metrics/slis` for apiserver.
```
Comment (laxmikantbpandhare) [inline releases/release-1.26/release-notes/maps/pr-112741-map.yaml:3]: `Exposed health check SLI metrics on `metrics/slis` for apiserver.`

File: releases/release-1.26/release-notes/maps/pr-112748-map.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+pr: 112748
+releasenote:
+  text: Lock `ServerSideApply` feature gate to true with the feature already being
```
Comment (laxmikantbpandhare) [inline releases/release-1.26/release-notes/maps/pr-112748-map.yaml:3]: Locked `ServerSideApply` feature gate to true with the feature already being

**Review by laxmikantbpandhare — CHANGES_REQUESTED** [review]

File: releases/release-1.26/release-notes/maps/pr-111093-map.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+pr: 111093
+releasenote:
+  text: Fixed bug in `kubectl rollout history` where only the latest revision was
+    displayed when a specific revision was requested and an output format was specified,
```
Comment (ramrodo) [inline releases/release-1.26/release-notes/maps/pr-111093-map.yaml:4]:
```suggestion
    displayed when a specific revision was requested and an output format was specified.
```

File: releases/release-1.26/release-notes/maps/pr-112607-map.yaml (hunk 1 above)
Comment (harshanarayana) [reply releases/release-1.26/release-notes/maps/pr-112607-map.yaml:3]: Using `considered` here actually makes the readability lot more complicated.  let me change this to `volume mount cleanup now considers only plugin directory and not the entire kubelet root`

File: releases/release-1.26/release-notes/maps/pr-112357-map.yaml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+pr: 112357
+releasenote:
+  text: 'kube-scheduler: add taints filtering logic consistent with `TaintToleration`
```
Comment (ramrodo) [inline releases/release-1.26/release-notes/maps/pr-112357-map.yaml:3]:
```suggestion
  text: 'kube-scheduler: added taints filtering logic consistent with `TaintToleration`
```

File: releases/release-1.26/release-notes/maps/pr-112357-map.yaml [hunk 6]
```diff
@@ -0,0 +1,4 @@
...
+    plugin for `PodTopologySpread` plugin'
```
Comment (ramrodo) [inline releases/release-1.26/release-notes/maps/pr-112357-map.yaml:4]:
```suggestion
    plugin for `PodTopologySpread` plugin.'
```

File: releases/release-1.26/release-notes/maps/pr-112545-map.yaml [hunk 7]
```diff
@@ -0,0 +1,4 @@
+pr: 112545
+releasenote:
+  text: API Server tracing root span name for opentelemetry is changed from `KubernetesAPI`
+    to `HTTP GET`
```
Comment (ramrodo) [inline releases/release-1.26/release-notes/maps/pr-112545-map.yaml:4]:
```suggestion
    to `HTTP GET`.
```

File: releases/release-1.26/release-notes/maps/pr-112732-map.yaml [hunk 8]
```diff
@@ -0,0 +1,3 @@
+pr: 112732
+releasenote:
+  text: 'kubeadm: support image repository format validation.'
```
Comment (ramrodo) [inline releases/release-1.26/release-notes/maps/pr-112732-map.yaml:3]:
```suggestion
  text: 'kubeadm: supported image repository format validation.'
```

File: releases/release-1.26/release-notes/maps/pr-112741-map.yaml (hunk 2 above)
Comment (harshanarayana) [reply releases/release-1.26/release-notes/maps/pr-112741-map.yaml:3]: /Ack

File: releases/release-1.26/release-notes/maps/pr-112748-map.yaml (hunk 3 above)
Comment (harshanarayana) [reply releases/release-1.26/release-notes/maps/pr-112748-map.yaml:3]: /Ack

File: releases/release-1.26/release-notes/maps/pr-112751-map.yaml [hunk 9]
```diff
@@ -0,0 +1,4 @@
+pr: 112751
+releasenote:
+  text: 'kubeadm: fixed a bug when performing validation on `ClusterConfiguration`
+    networking fields'
```
Comment (ramrodo) [inline releases/release-1.26/release-notes/maps/pr-112751-map.yaml:4]:
```suggestion
    networking fields.'
```

File: releases/release-1.26/release-notes/maps/pr-112797-map.yaml [hunk 10]
```diff
@@ -0,0 +1,4 @@
+pr: 112797
+releasenote:
+  text: 'kube-apiserver: the unused `--master-service-namespace` flag is deprecated
```
Comment (ramrodo) [inline releases/release-1.26/release-notes/maps/pr-112797-map.yaml:3]:
```suggestion
  text: 'kube-apiserver: the unused `--master-service-namespace` flag was deprecated
```

File: releases/release-1.26/release-notes/maps/pr-112732-map.yaml (hunk 8 above)
Comment (harshanarayana) [reply releases/release-1.26/release-notes/maps/pr-112732-map.yaml:3]: Makes it really hard to read. I would rather change this to `kubeadm: now supports image repository format validation.`

**Review by ramrodo — CHANGES_REQUESTED** [review]
Nice!
Just some comments related to past tense and add full stop at the end

**Review by ramrodo — APPROVED** [review]
/lgtm
Thanks!

**Pull request merged**